.travis.yml

```diff
@@ -21,6 +21,10 @@ dist: trusty
 
 sudo: false
 
+cache:
+  - directories:
+    - $HOME/openssl
+
```

Comment:
Sorry, those lines might be wrong. We might not have to add those lines.
I referred python's case [1]. But reading the cache document [2], it is still not clear for me.

* [1] https://github.com/python/cpython/blob/master/.travis.yml#L11
* [2] https://docs.travis-ci.com/user/caching